figures/S3.supplemental_figure3/scripts/S3.r

```diff
@@ -0,0 +1,1060 @@
+suppressPackageStartupMessages(suppressWarnings(library(ggplot2)))
+suppressPackageStartupMessages(suppressWarnings(library(dplyr)))
+suppressPackageStartupMessages(suppressWarnings(library(patchwork)))
+suppressPackageStartupMessages(suppressWarnings(library(cowplot)))
+suppressPackageStartupMessages(suppressWarnings(library(RcppTOML)))
+suppressPackageStartupMessages(suppressWarnings(library(pheatmap)))
+suppressPackageStartupMessages(suppressWarnings(library(lattice)))
+suppressPackageStartupMessages(suppressWarnings(library("RColorBrewer")))
+suppressPackageStartupMessages(suppressWarnings(library(gplots)))
+suppressPackageStartupMessages(suppressWarnings(library(ComplexHeatmap)))
+suppressPackageStartupMessages(suppressWarnings(library(ggplotify)))
+suppressPackageStartupMessages(suppressWarnings(library(arrow)))
+suppressPackageStartupMessages(suppressWarnings(library(scales)))
+suppressPackageStartupMessages(suppressWarnings(library(circlize)))
+
+# check if ggmagnify is installed, if not install it
+if (!requireNamespace("ggmagnify", quietly = TRUE)) {
```

Comment:
Would it make it easier if this was already installed in your environment? Not sure if this is possible so just posing as a question.

Reply:
It will not install via conda or pip sadly!